docs/UsersGuide.asciidoc

```diff
@@ -972,6 +972,16 @@ after a certain number of days. That duration can be adjusted by setting
 +untracked_assets_storage_duration+ in the +misc_limits+ section of the
 openQA config to the desired number of days.
 
+In less trivial cases where a common limit is not enough or certain assets
+need more fine-grained control, patterns based on the filename can be used.
+As with the default duration for untracked assets, the limit is set in days.
+[source,ini]
+--------------------------------------------------------------------------------
+[assets/storage_duration]
+xml = 50
```

Comment:
I find this a little unclear. `xml` is the pattern in this example? Perhaps really using a pattern would be more helpful to the reader.

Reply:
It is a pattern already. Unless you mean "something that looks like a pattern to someone not used to regular expressions".

Reply:
just as `''` is a string. It still makes sense to use `'example'` in documentation as it's the more representative example of its class :)

And you said patterns in the documentation, but now you're talking about regular expressions. That and the question if it's a `match` or a `search` is quite important to know if you want to make use of it. The example given appears to me a key referencing something else actually (possibly black magic).